snippets/formatNumber.md

```diff
@@ -0,0 +1,32 @@
+---
+title: formatNumber
+tags: date,math,string,intermediate
+---
+
+Returns a number in readable format.
```

Comment:
```suggestion
Returns a number with commas as thousand separators.
```

Reply:
If we are using `toLocaleString`. I think the description could be 

Returns a number with using the local number format order

its more generic. 
Also I will add examples with multiple locale, so that there's no confusion for someone, who doesn't know about it.